lib/bearerstrategy.js

```diff
@@ -34,6 +36,13 @@ const jws = require('jws');
  * signature is:
  *
  *     function(token, done) { ... }
+ * or
+ *     function(req, token, done) { ... }
```

Comment:
Who sets up this pattern? The general pattern in JS is to add optional/additional arguments to the *end* of the arguments list so that there's no ambiguity. Can we change this now, or is it breaking?